### DISCUSSION TITLE
Parallel gripper sdf

### DISCUSSION THREAD

**Discussion by chaiban** [bulletphysics/bullet3#3631] [General]
Hi, Does anyone know how to added parallel gripper to KUKA sdf file given in pybullet_env ?.

I tried to added a different gripper base to kuka arm and it works. But when I try to add parallel grippers using prismatic joints it does not seem to work. 

Does anyone have an idea on How to create parallel gripper sdf file ? 

Thank you!

**Comment by wxplovehlt** [comment]
> Hi, Does anyone know how to added parallel gripper to KUKA sdf file given in pybullet_env ?.
> 
> I tried to added a different gripper base to kuka arm and it works. But when I try to add parallel grippers using prismatic joints it does not seem to work.
> 
> Does anyone have an idea on How to create parallel gripper sdf file ?
> 
> Thank you!

I also want to solve this problem. you have achieve it?
